x/clp/types/keys.go

```diff
@@ -81,8 +75,9 @@ func GetDefaultPmtpParams() *PmtpParams {
 
 func GetDefaultLiquidityProtectionParams() *LiquidityProtectionParams {
 	return &LiquidityProtectionParams{
-		MaxRowanLiquidityThreshold:      sdk.MustNewDecFromStr("1000"),
+		MaxRowanLiquidityThreshold:      sdk.NewUint(1000),
```

Comment:
@joneskm does it mean we can use floating numbers anymore, what if we use BTC as a threshold asset where threshold value is set in BTC and if BTC price very high and you want to rely on decimals for more accuracy, would that means that would not be possible with current implementation?